### ISSUE TITLE
"Get Help" link points to "Page not Found"

### ISSUE THREAD

**Issue by thekcool** [MrAlex94/Waterfox#2964]
### What happened?

A bug happened!

The "Get Help" in "Help" and the "Waterfox Help" in the "About Waterfox" pop up link points to https://www.waterfox.net/support/WINNT/firefox-help which shows 

Page Not Found

We could not find what you were looking for.

Please contact the owner of the site that linked you to the original URL and let them know their link is broken.

### Reproducible?

- [ ] I have tested this on Firefox.

### Version

G5

### What platform are you seeing the problem on?

Windows

**Comment by Tiltowait2** [comment]
Yup, some genius broke the link on the web server without checking if it was in use.
